### ISSUE TITLE
F-Droid package isn't allowed to access "accessibility features" API on Android 13

### ISSUE THREAD

**Issue by gQUltvGetZej8Fpfe3FyswuMnR910t1tGVeVuR1** [bitwarden/mobile#2530]
### Steps To Reproduce

1. Add Bitwardens F-Droid repository to F-Droid
2. Install Bitwarden
3. Login to Bitwarden and access the settings
4. Try to register Bitwarden as accessibility feature

### Expected Result

Be allowed to add Bitwarden as accessibility feature. This should be allowed from apps installed from 3rd party app stores.

### Actual Result

Warning message from Android (Restricted setting)

### Screenshots or Videos

(German news board)
https://m.winfuture.de/news/129489

### Additional Context

Installing the PlayStore version fixed that behavior.

### Operating System

Android

### Operating System Version

13

### Device

Google Pixel 5

### Build Version

latest available 2023-05-21

### Beta

- [ ] Using a pre-release version of the application.

**Comment by micahblut** [comment]
We are not planning to implement the accessibility service in the upcoming native apps. As a result, I will close this issue.
